test/DurableTask.SqlServer.Tests/Integration/DatabaseManagement.cs

```diff
@@ -390,17 +539,20 @@ public IEnumerable<string> GetSchemas()
                 }
             }
 
-            public IEnumerable<string> GetTables()
+            public IEnumerable<string> GetTables(string schemaName = "dt")
             {
                 this.testDb.Tables.Refresh();
                 foreach (Table table in this.testDb.Tables)
                 {
                     // e.g. "dt.History"
-                    yield return $"{table.Schema}.{table.Name}";
+                    if(table.Schema == schemaName)
```

Comment:
code style nit: please add a space after the `if` here, and on lines 566, 583, and 600.